src/Microsoft.Health.Dicom.Functions.UnitTests/Update/UpdateDurableFunctionTests.Activity.cs

```diff
@@ -36,15 +36,15 @@ public async Task GivenInstanceMetadata_WhenUpdatingInstanceWatermark_ThenShould
 
         var versions = expected.Select(x => x.Version).ToList();
 
-        _indexStore.BeginUpdateInstancesAsync(Partition.DefaultKey, studyInstanceUid, CancellationToken.None).Returns(identifiers);
+        _indexStore.BeginUpdateInstancesAsync(Arg.Any<Partition>(), studyInstanceUid, CancellationToken.None).Returns(identifiers);
```

Comment:
nit: if this is the default partition, you should be able to pass it here as it will use reference equality on the type.